docs/tests/development/get_channels_nhsapp_accounts/400.md

```diff
@@ -0,0 +1,79 @@
+# 400 Tests
+
+These tests target the API endpoint GET /channels/nhsapp/account testing that 400 is returned when the ODS code is missing or invalid
+
+
+## Scenario: An API consumer submitting a GET NHS App Accounts request an invalid ODS Code
+
+**Given** the API consumer sends a request to get NHS App Accounts with an invalid ODS Code
+<br/>
+**When** the request is submitted
+<br/>
+**Then** the service responds with a 400 invalid response, telling the user the ODS Code is invalid
+<br/>
+
+**Asserts**
+- Response returns a 400 ‘Invalid’ error
+- Response returns the expected error message body
+- Response returns the ‘X-Correlation-Id’ header if provided
+
+**Methods**
+
+This test makes use of different HTTP methods, if the method is either HEAD or OPTIONS the test will not assert against the body of the response as none is returned.
+
+| Value   |
+|---------|
+| GET     |
+| POST    |
+| PUT     |
+| PATCH   |
+| DELETE  |
+| HEAD    |
+| OPTIONS |
+
+**Correlation IDs**
```

Comment:
That functionality doesn't exist right now does it?

Reply:
It does, it's common to all the endpoint of the proxy.

Reply:
I'm not sure about the BE though.

Reply:
Ah ok, haven't worked too close to the detail on this. I just know we have that ticket for using the correlation ID in the backend and didn't know if we'd specified it on the FE either